modules/simphera_common/simphera-instances.tf

```diff
@@ -1,10 +1,8 @@
 module "simphera_instance" {
   for_each = var.simpheraInstances
 
-  source = "./modules/simphera_instance"
+  source = "../simphera_instance"
```

Comment:
I am not sure if the logical structure of the modules should be reflected by the folder structure. Logically, simphera_instance is a submodule of common. Does it make sense to use simphera_instance independently from common?